scene/gui/graph_node.cpp

```diff
@@ -668,15 +668,15 @@ void GraphNode::_port_pos_update() {
 		if (slot_table.has(slot_index)) {
 			if (slot_table[slot_index].enable_left) {
 				PortCache port_cache;
-				port_cache.pos = Point2i(edgeofs, vertical_ofs + size.height / 2);
+				port_cache.pos = Point2i(edgeofs, vertical_ofs + size.height / 2).rotated(get_rotation());
```

Comment:
```suggestion
				port_cache.pos = Point2(edgeofs, vertical_ofs + size.height / 2).rotated(get_rotation());
```
Sorry I missed you missed this part too in my suggestion

Reply:
yes, so just to clarify, the correct is: 
Point2(edgeofs, vertical_ofs + size.height / 2).rotated(get_rotation());
and
Point2(get_size().width - edgeofs, vertical_ofs + size.height / 2).rotated(get_rotation());

Reply:
Yes, as my original suggestion, it should be, please just compile and confirm it yourself as well, I'm not testing this myself